.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # tag=v3.1.0
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+      - name: Install Helm
+        uses: azure/setup-helm@b5b231a831f96336bbfeccc1329990f0005c5bb1 # tag=v3.3
+        with:
+          version: v3.10.0 # renovate: datasource=github-releases depName=helm packageName=helm/helm
+
+      - name: Run chart-releaser
```

Comment:
Should we update the README with instructions to use it? Something like this:

## Helm

### Add Helm repository

```shell
helm repo add renovate-on-prem https://whitesource.github.io/renovate-on-prem
helm repo update
```

### Install Renovate chart

```shell
helm install --generate-name --set renovate.config='\{\"token\":\"...\"\}' renovate-on-prem/whitesource-renovate
```

> See the available [values](./helm-charts/whitesource-renvoate/values.yaml) for full configuration.

Reply:
Yep! Added [here](https://github.com/mend/renovate-on-prem/pull/234).